src/style.css

```diff
@@ -0,0 +1,135 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+body,
+h2,
+p {
+  margin: 0;
+  padding: 0;
+
+  --main-accent-color: #060b35;
+  --secondary-color: #616070;
+}
+
+.card {
+  box-sizing: border-box;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
```

Comment:
redundant styles. To avoid it - add `display:block` to your `<img>` and `<a>`
```suggestion

```

Reply:
not fixed

Reply:
is still not fixed

Reply:
Fixed